src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using System;
+using System.Resources;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Microsoft.Azure.Management.Sql")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+[assembly: AssemblyTitle("")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+
```

Comment:
Current version in master branch is 0.9.0.0.  Since this is a breaking change, I'm going with a major version bump 

---
In reply to: [84785739](https://github.com/Azure/azure-sdk-for-net/pull/2493#discussion_r84785739) [](ancestors = 84785739,84770771)